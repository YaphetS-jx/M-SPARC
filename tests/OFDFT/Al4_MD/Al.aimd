:MDSTEP: 1
:R:
  0.0000000000E+00   0.0000000000E+00   0.0000000000E+00
  4.0000000000E+00   0.0000000000E+00   4.0000000000E+00
  0.0000000000E+00   4.0000000000E+00   4.0000000000E+00
  4.0000000000E+00   4.0000000000E+00   0.0000000000E+00
:V:
 -1.0462861728E-04  -1.0134539488E-04  -1.9937370466E-04
  1.1333872889E-04   3.5725153487E-05   2.6967464944E-04
  9.2726447800E-06  -8.8889653375E-05  -5.1188248639E-05
 -1.7982756390E-05   1.5450989476E-04  -1.9112696138E-05
:F:
  2.4031201322E-16  -1.2988213422E-15   8.8422792969E-16
 -3.1348307634E-17   1.0793796949E-15   5.9514379207E-16
  3.9254942275E-16   2.2704981199E-16  -1.9182784474E-16
 -6.0151312834E-16  -7.6081646575E-18  -1.2875438770E-15
:TEL:   3.1577300000E+02
:TIO:   3.1577300000E+02
:TE:  -2.2119638485E+00
:MDSTEP: 2
:R:
  7.9956745093E+00   7.9958102422E+00   7.9917576172E+00
  4.0046855787E+00   1.4769269078E-03   4.0111487204E+00
  3.8334386967E-04   3.9963251797E+00   3.9978838075E+00
  3.9992565682E+00   4.0063876512E+00   7.9992098549E+00
:V:
 -1.0448386419E-04  -1.0121876776E-04  -1.9907923185E-04
  1.1318595626E-04   3.5725559437E-05   2.6931543267E-04
  9.3124912974E-06  -8.8841090653E-05  -5.1251974331E-05
 -1.8014583362E-05   1.5433429898E-04  -1.8984226486E-05
:F:
  3.4442902614E-04   3.0129963899E-04   7.0067578068E-04
 -3.6351092713E-04   9.6592726808E-07  -8.5472914073E-04
  9.4811774983E-05   1.1555132531E-04  -1.5163046415E-04
 -7.5729873991E-05  -4.1781689157E-04   3.0568382420E-04
:TEL:   3.1497782342E+02
:TIO:   3.1497782342E+02
:TE:  -2.2119647847E+00
:MDSTEP: 3
:R:
  7.9913609871E+00   7.9916309543E+00   7.9835395823E+00
  4.0093585257E+00   2.9538873807E-03   4.0222677397E+00
  7.6998235883E-04   3.9926543746E+00   3.9957623460E+00
  3.9985105048E+00   4.0127607837E+00   7.9984303320E+00
:V:
 -1.0411824814E-04  -1.0090204369E-04  -1.9833151421E-04
  1.1280099910E-04   3.5737649105E-05   2.6841139084E-04
  9.4247543848E-06  -8.8734911198E-05  -5.1442621322E-05
 -1.8107505341E-05   1.5389930578E-04  -1.8637255312E-05
:F:
  5.2552672143E-04   4.5232134333E-04   1.0784618165E-03
 -5.5246552636E-04   2.7800524027E-05  -1.2963700281E-03
  1.7230975255E-04   1.3709465773E-04  -3.0199963420E-04
 -1.4537094762E-04  -6.1721652509E-04   5.1990784574E-04
:TEL:   3.1299217067E+02
:TIO:   3.1299217067E+02
:TE:  -2.2119661766E+00
:MDSTEP: 4
:R:
  7.9870657265E+00   7.9874673841E+00   7.9753590229E+00
  4.0140122751E+00   4.4318138955E-03   4.0333417114E+00
  1.1626084489E-03   3.9889883335E+00   3.9936303903E+00
  3.9977593899E+00   4.0191124686E+00   7.9976688754E+00
:V:
 -1.0362834085E-04  -1.0048257366E-04  -1.9732422825E-04
  1.1228660557E-04   3.5770620340E-05   2.6720560189E-04
  9.5927119846E-06  -8.8617745439E-05  -5.1742920408E-05
 -1.8250976705E-05   1.5332969876E-04  -1.8138453231E-05
:F:
  6.4017063869E-04   5.4577586762E-04   1.3182989370E-03
 -6.7149497953E-04   5.0652035448E-05  -1.5727135897E-03
  2.2733265545E-04   1.4169240727E-04  -4.1253933427E-04
 -1.9600831460E-04  -7.3812031034E-04   6.6695398700E-04
:TEL:   3.1037122540E+02
:TIO:   3.1037122540E+02
:TE:  -2.2119659393E+00
:MDSTEP: 5
:R:
  7.9827927113E+00   7.9833227791E+00   7.9672242731E+00
  4.0186426907E+00   5.9115005206E-03   4.0443610328E+00
  1.5631341338E-03   3.9853272160E+00   3.9914840992E+00
  3.9970014639E+00   4.0254385044E+00   7.9969305949E+00
:V:
 -1.0298629653E-04  -9.9935426136E-05  -1.9600459617E-04
  1.1161319282E-04   3.5818778156E-05   2.6562960093E-04
  9.8173470087E-06  -8.8471436998E-05  -5.2148522380E-05
 -1.8444243290E-05   1.5258808498E-04  -1.7476482375E-05
:F:
  8.8752525354E-04   7.5612029683E-04   1.8216658176E-03
 -9.3083974163E-04   6.3935847421E-05  -2.1772615403E-03
  3.0716939798E-04   2.0643746325E-04  -5.5255988582E-04
 -2.6385490989E-04  -1.0264936075E-03   9.0815560853E-04
:TEL:   3.0696953357E+02
:TIO:   3.0696953357E+02
:TE:  -2.2119647903E+00
