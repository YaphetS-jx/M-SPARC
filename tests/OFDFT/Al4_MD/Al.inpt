# Test: Al #
CELL: 8 8 8
MESH_SPACING: 0.5
FD_ORDER: 6
BC: P P P
EXCHANGE_CORRELATION: LDA_PZ

TOL_PSEUDOCHARGE: 1.00E-08

OFDFT_FLAG: 1
TOL_OFDFT: 1E-4
OFDFT_LAMBDA: 0.2

MD_FLAG: 1
MD_METHOD: NVE
MD_TIMESTEP: 1
MD_NSTEP: 5
ION_TEMP: 315.773
ION_ELEC_EQT: 1 
PRINT_RESTART_FQ: 5
RESTART_FLAG: 0