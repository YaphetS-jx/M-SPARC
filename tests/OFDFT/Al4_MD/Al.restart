:MDSTEP: 5
:R:
  7.9827970550E+00   7.9833271055E+00   7.9672324748E+00
  4.0186379621E+00   5.9097309660E-03   4.0443497436E+00
  1.5624582712E-03   3.9853313177E+00   3.9914869035E+00
  3.9970025246E+00   4.0254318459E+00   7.9969308781E+00
:V:
 -1.0288273022E-04  -9.9833992489E-05  -1.9580728559E-04
  1.1150078302E-04   3.5781380375E-05   2.6536142618E-04
  9.8062534549E-06  -8.8380704815E-05  -5.2093634302E-05
 -1.8424306250E-05   1.5243331693E-04  -1.7460506288E-05
:TEL:   3.0634908732E+02
:TIO:   3.0634908732E+02
