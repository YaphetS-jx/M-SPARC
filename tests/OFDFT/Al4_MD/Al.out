***************************************************************************
*                      M-SPARC v1.0.0 (Mar 29, 2021)                      *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 12-Apr-2021  Start time: 11:27:07                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 8.000000 8.000000 8.000000 
FD_GRID: 16 16 16
FD_ORDER: 12
BC: P P P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
EXCHANGE_CORRELATION: LDA_PZ
OFDFT_FLAG: 1
OFDFT_LAMBDA: 0.200000
TOL_OFDFT: 1.00E-04
CALC_STRESS: 0
CALC_PRES: 0
MD_FLAG: 1
MD_METHOD: NVE
MD_TIMESTEP: 1.00
MD_NSTEP: 5
ION_ELEC_EQT: 1
ION_TEMP: 315.773000
TOL_PSEUDOCHARGE: 1.00E-08
REFERENCE_CUTOFF: 0.50
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_DENSITY: 0
PRINT_MDOUT: 1
PRINT_VELS: 1
PRINT_RESTART: 1
PRINT_RESTART_FQ: 5
OUTPUT_FILE: ../tests/OFDFT/Al4_MD/Al.out
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing                       :  0.500000 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  ../tests/OFDFT/Al4_MD/Al.out
Total number of atom types         :  1
Total number of atoms              :  4
Total number of electrons          :  12
Atom type 1  (valence electrons)   :  Al 3
Pseudopotential                    :  Al_goodwin.psp8
lloc                               :  0
Atomic mass                        :  26.981538499999999
Pseudocharge radii of atom type 1  :  11.00 11.00 11.00
Number of atoms of type 1          :  4
Estimated total memory usage       :  13.06 MB
=====================================================================
                Orbital-Free DFT NLCG (OFDFT-NLCG#1)                
=====================================================================
Iteration      Free Energy (Ha/atom)   NLCG Error        Timing (sec)
1              -2.2114653075E+00       1.125E-02         0.011
2              -2.2123445996E+00       7.638E-03         0.009
3              -2.2130046001E+00       2.992E-03         0.009
4              -2.2130643674E+00       1.453E-03         0.008
5              -2.2130835219E+00       6.365E-04         0.008
6              -2.2130872610E+00       3.770E-04         0.009
7              -2.2130883781E+00       1.787E-04         0.008
8              -2.2130886748E+00       1.226E-04         0.009
9              -2.2130888409E+00       7.569E-05         0.008
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -2.2130888409E+00 (Ha/atom)
Total free energy                  : -8.8523553636E+00 (Ha)
Kinetic energy                     :  3.3038795150E+00 (Ha)
Exchange correlation energy        : -3.1454956628E+00 (Ha)
Self and correction energy         : -1.6403679287E+01 (Ha)
Average force                      :  1.1840049699E-15 (Ha/Bohr)
Maximum force                      :  1.5895111746E-15 (Ha/Bohr)
Time for force calculation         :  0.476 (sec)
Relax time                         :  0.939 (sec)
=====================================================================
                Orbital-Free DFT NLCG (OFDFT-NLCG#2)                
=====================================================================
Iteration      Free Energy (Ha/atom)   NLCG Error        Timing (sec)
1              -2.2130850150E+00       3.655E-04         0.007
2              -2.2130859396E+00       2.677E-04         0.007
3              -2.2130867122E+00       1.462E-04         0.006
4              -2.2130869442E+00       9.588E-05         0.006
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -2.2130869442E+00 (Ha/atom)
Total free energy                  : -8.8523477768E+00 (Ha)
Kinetic energy                     :  3.3036667661E+00 (Ha)
Exchange correlation energy        : -3.1454424549E+00 (Ha)
Self and correction energy         : -1.6403668688E+01 (Ha)
Average force                      :  6.2545465450E-04 (Ha/Bohr)
Maximum force                      :  9.2881808293E-04 (Ha/Bohr)
Time for force calculation         :  0.420 (sec)
Relax time                         :  0.790 (sec)
=====================================================================
                Orbital-Free DFT NLCG (OFDFT-NLCG#3)                
=====================================================================
Iteration      Free Energy (Ha/atom)   NLCG Error        Timing (sec)
1              -2.2130786681E+00       4.572E-04         0.007
2              -2.2130799976E+00       2.644E-04         0.006
3              -2.2130808728E+00       1.797E-04         0.006
4              -2.2130811313E+00       1.184E-04         0.007
5              -2.2130812619E+00       5.130E-05         0.007
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -2.2130812619E+00 (Ha/atom)
Total free energy                  : -8.8523250476E+00 (Ha)
Kinetic energy                     :  3.3036893026E+00 (Ha)
Exchange correlation energy        : -3.1454539943E+00 (Ha)
Self and correction energy         : -1.6403633881E+01 (Ha)
Average force                      :  9.7133262088E-04 (Ha/Bohr)
Maximum force                      :  1.4094560215E-03 (Ha/Bohr)
Time for force calculation         :  0.450 (sec)
Relax time                         :  0.857 (sec)
=====================================================================
                Orbital-Free DFT NLCG (OFDFT-NLCG#4)                
=====================================================================
Iteration      Free Energy (Ha/atom)   NLCG Error        Timing (sec)
1              -2.2130713500E+00       1.554E-04         0.010
2              -2.2130715437E+00       1.191E-04         0.007
3              -2.2130716870E+00       7.263E-05         0.007
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -2.2130716870E+00 (Ha/atom)
Total free energy                  : -8.8522867480E+00 (Ha)
Kinetic energy                     :  3.3039320464E+00 (Ha)
Exchange correlation energy        : -3.1455245841E+00 (Ha)
Self and correction energy         : -1.6403576184E+01 (Ha)
Average force                      :  1.1951192591E-03 (Ha/Bohr)
Maximum force                      :  1.7108182754E-03 (Ha/Bohr)
Time for force calculation         :  0.453 (sec)
Relax time                         :  0.874 (sec)
=====================================================================
                Orbital-Free DFT NLCG (OFDFT-NLCG#5)                
=====================================================================
Iteration      Free Energy (Ha/atom)   NLCG Error        Timing (sec)
1              -2.2130578588E+00       2.087E-04         0.009
2              -2.2130581116E+00       1.393E-04         0.007
3              -2.2130584189E+00       9.089E-05         0.008
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -2.2130584189E+00 (Ha/atom)
Total free energy                  : -8.8522336756E+00 (Ha)
Kinetic energy                     :  3.3040698881E+00 (Ha)
Exchange correlation energy        : -3.1455775539E+00 (Ha)
Self and correction energy         : -1.6403495831E+01 (Ha)
Average force                      :  1.6480946270E-03 (Ha/Bohr)
Maximum force                      :  2.3687587957E-03 (Ha/Bohr)
Time for force calculation         :  0.477 (sec)
Relax time                         :  0.870 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  4.598 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
